From 1e73cedda4b86f29610887977232295d0941bb50 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Thu, 15 Aug 2024 10:08:48 +0300
Subject: [PATCH] [SW-15774] pack: preboot: treat warning as error

The -Werror was missed out during the "split" procedure.
Add the -Werror to the compilation flags.

Change-Id: Ibaf620843f0b964dd5fa69038b6f547e1397095d
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/package/preboot/preboot.mk b/package/preboot/preboot.mk
index 305029f..ed6e8a4 100644
--- a/package/preboot/preboot.mk
+++ b/package/preboot/preboot.mk
@@ -63,6 +63,7 @@
 PREBOOT_CONF_OPTS += -DCMAKE_EXE_LINKER_FLAGS_INIT=-Wl,-e__start
 PREBOOT_CONF_OPTS += -DTARGET=$(HL_TARGET)
 PREBOOT_CONF_OPTS += -DEXTRA_CFLAGS="\
+		-Werror \
 		-Wno-misleading-indentation \
 		$(PREBOOT_CONF_APP_EXTRA_CFLAGS)"
 PREBOOT_CONF_OPTS += $(PREBOOT_CONF_APP_OPTS)
